.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   docker-executor:
     docker:
-      - image: cimg/openjdk:8.0.322
+      - image: cimg/openjdk:8.0.322   # Version for Gradle 5.1 should be 11.0.18?
```

Comment:
I think the comment is outdated. We use Gradle 8.3 now in all three project (Roddy, BE, RTL). I will remove the comment. But I will also leave it as is at 8.0.322, because it works. I tried changing it before, but did not have time to figure out why it then broke. 

OK for you?